app/views/shared/_navbar.html.erb

```diff
@@ -12,16 +12,15 @@
     <ul class="navbar-nav mr-auto">
       <% if user_signed_in? %>
         <li class="nav-item active">
-          <%= link_to "Home", "#", class: "nav-link" %>
+          <%= link_to "All equipment", equipment_index_path, class: "nav-link" %>
         </li>
         <li class="nav-item">
-          <%= link_to "Messages", "#", class: "nav-link" %>
+          <%= link_to "My bookings", "#", class: "nav-link" %>
```

Comment:
perhaps we could put the My Bookings in the dropdown too ... ?

Reply:
i added it~